server/container_create_linux.go

```diff
@@ -477,6 +477,20 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 				return nil, err
 			}
 		}
+
+		if securityContext.NoNewPrivs {
+			const sysAdminCap = "CAP_SYS_ADMIN"
+			for _, cap := range specgen.Config.Process.Capabilities.Bounding {
+				if cap == sysAdminCap {
+					log.Warnf(ctx, "Setting `noNewPrivileges` flag has no effect because container has %s capability", sysAdminCap)
```

Comment:
Just a nit, so probably nothing to do here.

Would the flag name, and I assume we mean the one passed down from Kubernetes, be `noNewPrivs`?  I think the flag name is `no_new_privs` internally, as defined in the API, per the Protocol Buffer schema.

Reply:
Yeah I'm leaning towards using the runtime spec flag: https://github.com/opencontainers/runtime-spec/blob/99235413e05f23548c28dae02d72532e092d87a2/specs-go/config.go#L86

Reply:
@saschagrunert, ahh! Makes sense. :+1: